static/js/apps/explore_landing/topics.json

```diff
@@ -743,61 +743,61 @@
           },
           {
             "title": "SDG goals on gender equality across the world",
-            "url": "oq=SDG+goals+on+gender+equality+across+the+world&dc=sdg"
+            "url": "q=SDG+goals+on+gender+equality+across+the+world&dc=sdg"
           },
           {
             "title": "Women in managerial positions in India",
-            "url": "oq=Women+in+managerial+positions+in+India&dc=sdg"
+            "url": "q=Women+in+managerial+positions+in+India&dc=sdg"
           },
           {
             "title": "Access to clean drinking water in Mexico",
-            "url": "oq=Access+to+clean+drinking+water+in+Mexico&dc=sdg"
+            "url": "q=Access+to+clean+drinking+water+in+Mexico&dc=sdg"
           },
           {
             "title": "Access to primary school education in Afghanistan",
-            "url": "oq=Access+to+primary+school+education+in+Afghanistan&dc=sdg"
+            "url": "q=Access+to+primary+school+education+in+Afghanistan&dc=sdg"
           },
           {
             "title": "How is the world combating unregulated fishing?",
-            "url": "oq=How+is+the+world+combating+unregulated+fishing?&dc=sdg"
+            "url": "q=How+is+the+world+combating+unregulated+fishing?&dc=sdg"
           },
           {
             "title": "How has access to electricity improved in Kenya?",
-            "url": "oq=How+has+access+to+electricity+improved+in+Kenya?&dc=sdg"
+            "url": "q=How+has+access+to+electricity+improved+in+Kenya?&dc=sdg"
           },
           {
             "title": "How has informal employment changed over time in Bangladesh?",
-            "url": "oq=How+has+informal+employment+changed+over+time+in+Bangladesh?&dc=sdg"
+            "url": "q=How+has+informal+employment+changed+over+time+in+Bangladesh?&dc=sdg"
           },
           {
             "title": "How much food goes wasted around the world?",
-            "url": "oq=How+much+food+goes+wasted+around+the+world?&dc=sdg"
+            "url": "q=How+much+food+goes+wasted+around+the+world?&dc=sdg"
           }
         ],
         "comparison": [
           {
             "title": "Compare progress on poverty in Mexico, Nigeria and Pakistan",
-            "url": "oq=Compare+progress+on+poverty+in+Mexico,+Nigeria+and+Pakistan&dc=sdg"
+            "url": "q=Compare+progress+on+poverty+in+Mexico,+Nigeria+and+Pakistan&dc=sdg"
           },
           {
             "title": "Compare SDG goals on climate change between Brazil, China and India",
-            "url": "oq=Compare+SDG+goals+on+climate+change+between+Brazil,+China+and+India&dc=sdg"
+            "url": "q=Compare+SDG+goals+on+climate+change+between+Brazil,+China+and+India&dc=sdg"
           },
           {
             "title": "Violence vs poverty across the world",
-            "url": "oq=Violence+vs+poverty+across+the+world&dc=sdg"
+            "url": "q=Violence+vs+poverty+across+the+world&dc=sdg"
```

Comment:
@pradh any links with oq we have distriubuted?

Reply:
it might be, not sure.  if its a one-liner, map oq to q?

Reply:
sure